drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -107,8 +109,14 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     has_score = True
 
     def _(self, text):
-        """ Translate text """
-        return self.runtime.service(self, "i18n").ugettext(text)
```

Comment:
^ Am I right that the older, simpler code seen here is actually all that's going to be needed, in light of the latest changes to https://github.com/edx/edx-platform/pull/11575 ?